test/e2e/specs/test.spec.js

```diff
@@ -45,15 +95,37 @@ describe('Vaults UI Test Cases', () => {
         cy.contains(
           'p',
           'You can manage your vaults from the "My Vaults" view.',
+          { timeout: MINUTE_MS },
         ).should('exist');
+        cy.contains('Manage my Vaults').click();
       });
     });
 
-    it('should adjust the collateral by performing a withdrawl and approve the transaction successfully', () => {
-      cy.contains('button', 'Manage my Vaults').click();
-      cy.contains('button', 'Back to vaults').click();
-      cy.contains('div', /ATOM.*#8/).click();
+    it('should open the new vault', () => {
+      cy.get('span')
```

Comment:
I think since we are testing for adjusting collateral, we can move the code in this test case to the subsequent test case. Or if you think we will reuse this code at places, perhaps we create a custom command for it?

Reply:
we arent reusing the code at any other place and it is only being triggered once. can you explain further what you mean?

Reply:
I am suggesting that we should not have this as a separate test case. We are not testing `should open the new vault`. Instead we are testing `should adjust the collateral by performing a withdrawl and approve the transaction successfully`.